src/components/TopicLoader/TopicLoader.js

```diff
@@ -148,6 +150,8 @@ class TopicLoader extends Component {
       return;
     }
 
+    matomo.trackEvent({ category: activeTopic.name, action: 'load' });
```

Comment:
if the creation of matomo object failed because there is no MATOMO_XX variables defined. what's happening here ?